src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,177 @@
+import logging
+from common.requester import DelayedRequester
+from common.storage.image import ImageStore
+
+LIMIT = 100
```

Comment:
I think a limit of 1000 (as in the old original) would be better here.  It seems to work pretty well that way.